SB2018/2018-04-12.md

```diff
@@ -0,0 +1,56 @@
+---
+to: All Teams
+subject: Rules amendments
+---
+
+Hi!
+
+Some issues have come to light with [our rules][rules], and we've made an amendment:
+
+## Change 1: Token Marker ID changed
+
+We have changed the IDs of marker tokens, as defined in section 3.1. This is of relevance to your team if they have hard-coded the IDs of the tokens from the rules, instead of the recommended way of using the API. There's no need to update your robots to have the new numbers, they are already correct.
```

Comment:
I suggest you state that you don't need to update if youre using the recommended consts. If you're hardcoding the IDs, then you still will.